NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Em verb logic

## Link to Ticket

Closes #11 

## Description of Proposed Changes

Adds a new URL on the API Root that will display a list of inactive customers

## Steps to Test

```sh
git fetch --all
git checkout EM_VerbLogic
```

1. Start your virtual environment and server
1. Navigate to http://127.0.0.1:8000/bangazonapi/
1. Select the Users view and make sure there are a few users who have not logged in in over a year, and a few who have logged in during the last year.
1. Back on the API root, click on the "inactive_customers" link
1. Verify that the inactive customers you created in the data are the only ones who appear on the screen

## Impacted Areas in Application

**Modified**

faker_factory.py
models/Cust_Order.py
models/Employment_Dates.py
models/Ordered_Products.py
models/Payment_Options.py
models/User.py
serializers/Cust_order_Serializer.py
serializers/Employment_Dates_Serializer.py
serializers/Ordered_Products_Serializer.py
serializers/Payment_Options_Serializer.py
views/__init__.py
serializers/__init__.py
bangazonapi/urls.py

**New**

serializers/Inactive_User_Serializer.py
views/Inactive_Customer_View.py

## Mentions @username

@obscure-shadow @JacobAndrewSmith92 @pwmurphy89
